src/mcts/node.h

```diff
@@ -264,7 +264,8 @@ class Node {
   // subtree. For terminal nodes, eval is stored. This is from the perspective
   // of the player who "just" moved to reach this position, rather than from the
   // perspective of the player-to-move for the position.
-  float q_ = 0.0f;
+  // Initialized to loss (-1).
+  float q_ = -1.0f;
```

Comment:
I think the code assumes we're running in FpuStrategy = absolute, with FpuValue = -1.0. Let's use GetFpuValue here?

For FpuStrategy = reduction I don't know what to do, will capturing the GetFpuValue at the time the node is constructed work? Or just punt on that when this is enabled? I think most testers are still using this mode right?

Reply:
I clopped and tested this with FPU reduction since it's stronger in play. There isn't really any reason why this should use the same FPU method for initial Q value. FPU decides the 0 visit value and this only affects the value after node has visits. Option name could probably modified to make this more clear.

I did test a version where I modified `GetQ(fpu)` to set the `q_` to fpu if node doesn't have any visits. That version was weaker in testing: `Score of lc0_fpu_visits_fpu_init_35258 vs lc0_fpu_visits_35258: 26 - 42 - 332  [0.480] 400
Elo difference: -13.90 +/- 13.98, LOS: 2.62 %, DrawRatio: 83.0 %`.

Reply:
Ok I was thinking of this as treating the FPU value as a sort of Bayesian prior on q_, which gets mixed into q_. Instead they are considered different, I don't have a good idea for a name right now but maybe some comment here in the code that q_ = -1.0f is not the same as the FPU?

Reply:
It could be named something like "value_initialization"
I was thinking also that fpu_visit is maybe not the right term for the divisor. I guess something like value_divisor_init instead.

Reply:
value_pessimism?